_src/html/pages/contact.html

```diff
@@ -6,46 +6,58 @@
 <section>
 	<div class="edged-section">
 		<h1 class="page-title">Get in touch</h1>
-		<p class="page-title-subtext">If you think we might be able to help you achieve your business goals or are interested in finding out more, please get in touch. We’d be happy to see what we could do for you.</p>		
+		<p class="page-title-subtext">If you think we can help you achieve your business goals or are interested in finding out more, please get in touch. We’d be happy to see what we could do for you.</p>		
 	</div>
 </section>
 
 
 <section class="default-bg-section">
 	<div class="edged-section">
+
 		<div class="contact-details">
 			<div class="contact-details__item">
-				<h2 class="contact-details__label">Email us</h2>
-				<p class="contact-details__info"><a href="mailto:info@aqovia.com">info@aqovia.com</a></p>
+				<h2 class="contact-details__label">International Sales Manager</h2>
+				<p class="contact-details__info">Robert Wolters</p>
+				<p class="contact-details__info u-smaller-text"><a href="mailto:robert.wolters@aqovia.com">robert.wolters@aqovia.com</a></p>
```

Comment:
paring Rob from being spammed by e-mail harvesting crawlers and by using an obfuscation library to hid the mailto: link. Plenty of them here: https://www.npmjs.com/search?q=obfuscate%20email

Reply:
It seems that because the changes are requires ASAP, we are not looking into obfuscating e-mails right now and a decision has been made to replace Robert's mail with the one that was already there (info@aqovia.com)

Reply:
Okay, if we don't have time to install an NPM package. Just stick his e-mail address in here and copy/paste the obfuscated output HTML: https://www.albionresearch.com/misc/obfuscator.php

(Shouldn't take more than 2 mins.)

I'm not sure if anyone actually reads info@aqovia.com.